### PR TITLE
fix gradient border in email button

### PR DESCRIPTION
**Pull request by upacyxou**
- gradient border returned to "submit email" button

### CODE REVIEW

**Review by T-Damer — APPROVED** [review]

File: src/components/EmailForm.tsx [hunk 1]
```diff
@@ -30,19 +31,21 @@ export default function ({
...
-        fullWidth
-        center
-        small={submitType !== 'primary'}
-        type={submitType}
-        disabled={!emailIsValid}
-        onClick={() => onSubmit(email)}
-      >
-        {submitText}
-      </Button>
+      <GradientButton>
```
Comment (smyzin) [inline src/components/EmailForm.tsx:34]: Maybe we should name it `GradientBorder` instead?

File: src/components/GradientButton.tsx [hunk 2]
```diff
@@ -10,7 +10,6 @@ import classnames, {
 } from 'classnames/tailwind'
 
 const parentButtonWrapper = classnames(
-  width('w-fit'),
```
Comment (smyzin) [inline src/components/GradientButton.tsx:13]: This style makes it fit the button on small screens. May you test that everything is fine, where that component is used?

Comment (upacyxou) [reply src/components/GradientButton.tsx:13]: sure, I tested it. The only two places where we're using it is CTA Buttons and "Submit emal" button, I tested them both and it's works fine on any resolution up to 280px width

File: src/components/EmailForm.tsx (hunk 1 above)
Comment (upacyxou) [reply src/components/EmailForm.tsx:34]: I think you're right, it'll have more meaning. I'll rename it, ty

**Review by smyzin — APPROVED** [review]
Lgtm

**Pull request merged**